### PR TITLE
feat: use docarray package and remove labeler

### PR DESCRIPTION
**Pull request by tadejsv**
This replaces `DocumentArray` from `jina` with the one from `docarray`.

Also, the **labeler** and `DocumentArrayMemmap` ***are removed***.

Main changes, apart from switching imports:
- simplify `DocumentSequence` into simply `DocumentArray`
- replace a lot of `blob` attributes with `tensor`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
     long_description_content_type='text/markdown',
     zip_safe=False,
     setup_requires=['setuptools>=18.0', 'wheel'],
-    install_requires=['jina>=2.4.9', 'matplotlib'],
+    install_requires=['jina>=2.6.5.dev60', 'docarray>=0.4.2'],
```
Comment (bwanglzu) [inline setup.py:42]: why `jina` is required?

**Review by bwanglzu — CHANGES_REQUESTED** [review]
shall we also remove `https://github.com/jina-ai/jina/archive/refs/heads/master.zip` from `requirements-cicd(gpu)`?

File: setup.py (hunk 1 above)
Comment (tadejsv) [reply setup.py:42]: Ah good point, forgot to remove

**Review by bwanglzu — APPROVED** [review]
LGTM!

**Pull request merged**